src/xunit.performance.api/XunitRunner.cs

```diff
@@ -4,108 +4,95 @@
 
 using System;
 using System.Threading;
-using System.Reflection;
 using Xunit.Runners;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
 
 namespace Microsoft.Xunit.Performance.Api
 {
-    public class XunitRunner
+    public static class XunitRunner
     {
-        // We use consoleLock because messages can arrive in parallel, so we want to make sure we get
-        // consistent console output.
-        private object _consoleLock = new object();
-
-        // Use an event to know when we're done
-        private ManualResetEvent _finished = new ManualResetEvent(false);
-
-        // Start out assuming success; we'll set this to 1 if we get a failed test
-        private int _result = 0;
-
-        public int Run(string assemblyPath)
+        public static int Run(string assemblyPath, string typeName = null)
         {
+            if(typeName != null && string.IsNullOrWhiteSpace(typeName))
+                throw new ArgumentException("The test type name cannot be white space.");
+
             // Create a runner for the specifid assembly.
-            using(AssemblyRunner runner = AssemblyRunner.WithoutAppDomain(assemblyPath))
+            using (var runner = AssemblyRunner.WithoutAppDomain(assemblyPath))
```

Comment:
I think when you see the type right there (e.g. new <your type here>()), then it's not a big deal.  For other places where a method is called, (like line 27), we should have the type name present since it's not clear what the return type is.  What do you think?